### PR TITLE
Reenable API check

### PR DESCRIPTION
**Pull request by JunTaoLuo**
#617 Unblocked by https://github.com/aspnet/BuildTools/commit/d944172721e77b6ca05a512303a0859634786746

### CODE REVIEW

**Comment by pakrym** [comment]
Nope, still fails

**Comment by JunTaoLuo** [comment]
API check seems to work for Kestrel, why is IIS different?

**Comment by pakrym** [comment]
Are netcoreapp versions the same?

File: .appveyor.yml [hunk 1]
```diff
@@ -2,9 +2,8 @@ init:
   - git config --global core.autocrlf true
 branches:
   only:
-    - master
-    - release
     - dev
+    - /^release\/.*$/
```
Comment (javiercn) [inline .appveyor.yml]: This need to be on the release/2.1 branch I believe, before appveyor/travis will discover it.

**Comment by JunTaoLuo** [comment]
Sigh... our CI's publish step didn't complete correctly so not all packages were uploaded to aspnetcore-release. This means the packages on the feed is incoherent and this PR won't pass. Will try again tomorrow.

**Pull request merged**